src/main/resources/file-download-context.xml

```diff
@@ -40,30 +40,34 @@
 	
 
 	<util:list id="fileRetrieverFilter" value-type="java.lang.String">
-		<value>OrthologsCSVFromZinc</value>
- 		<value>UniProtToKEGG</value>
-		<value>FlyBaseToUniprotReferenceDNASequence</value>
- 		<value>OrphanetToUniprotReferenceDNASequence</value>
-		<value>Zinc</value>
-		<value>OrthologsFromZinc</value>
-		<value>RheaRetriever</value>
+		<value>IntEnz</value>		
+		<value>UniProtToENSEMBL</value>
 		<value>EnsemblToALL</value>
-		<value>UniProtToOMIM</value>
+		<value>OpenTargets</value>
+		<value>FlyBaseToUniprotReferenceDNASequence</value>
+		<value>CTD</value>
 		<value>UniProtToEntrez_Gene</value>
-		<value>UniProtToPDB</value>
+		<value>UniProtToUCSC</value>
```

Comment:
Just curious:  Why have `UniProtToENSEmBL` and `EntrezGeneToUniprotReference` been removed and `UniProtToUCSC` added?

Reply:
I wasn't the one who actually made that change. Basically, 'file-download-context.xml' and 'create-references-context.xml' were older config files that were originally used to run either the download or insertion steps in add-links. They're basically the download/insertion parts of 'application-context.xml', split up. Since we've used the Perl wrapper to do this over the last year to comment out application-context, these two files became out of sync due to disuse. For Jenkins, I am reviving their use and I just copied the pertinent parts from application-context to update them.

In short, this is actually a change that probably happened to add-links in the last year or so, but I don't know for sure

Reply:
UniProtToENSEMBL was moved from line 56 to 44, not removed. From what I can tell, `EntrezGeneToUniprotReference` was never used, however file-download-context.xml is probably old enough that the reference to that bean still existed. It has not been in the main application-context.xml for a while.